pkg/sip/room.go

```diff
@@ -181,8 +191,41 @@ func (r *Room) SetOutput(out media.Writer[media.PCM16Sample]) {
 	r.out.Set(media.ResampleWriter(out, r.mix.SampleRate()))
 }
 
+func (r *Room) SetDTMFOutput(out *rtp.Stream) {
+	if r == nil {
+		return
+	}
+	if out == nil {
+		r.outDtmf.Store(nil)
+		return
+	}
+	r.outDtmf.Store(out)
```

Comment:
Do we need to single out the `out == nil` case here?

Reply:
It's possible in this particular case, but I also use interface type in one similar place. Setting typed nil pointer to interface will lead to non-nil value that will panic when any method is called. So I'm being cautious here, just in case I will change the type to an interface later.